bedrock/settings/static_media.py

```diff
@@ -349,8 +343,6 @@
     },
     'firefox_fx38_0_5_firstrun': {
         'source_filenames': (
-            'css/sandstone/sandstone.less',
-            'css/tabzilla/tabzilla-static.less',
```

Comment:
Feel free to do whatever is easier here (which may be just adding `tabzilla-static.less` to this page-specific bundle). I don't want to make us add any more high level decisions to this already large PR. We can re-think when we tackle [bug 1186605](https://bugzilla.mozilla.org/show_bug.cgi?id=1186605) if necessary.